README.md

```diff
@@ -162,7 +162,7 @@ A Kubernetes incubator project, [cri-tools](https://github.com/kubernetes-sigs/c
 For async communication and long-running discussions please use issues and pull requests on the GitHub repo.
 This will be the best place to discuss design and implementation.
 
-For sync communication catch us in the `#containerd` and `#containerd-dev` Slack channels on Cloud Native Computing Foundation's (CNCF) Slack - `cloud-native.slack.com`. Everyone is welcome to join and chat. [Get Invite to CNCF Slack.](https://slack.cncf.io)
```

Comment:
https://slack.cncf.io seems still active

Reply:
Accessing "https://slack.cncf.io/" in my area is particularly slow, while "https://cloud-native.slack.com/" is much faster.

Reply:
I don't find any invitation page in <https://cloud-native.slack.com/>.
"Sign In With Email" button doesn't work with an unregistered address. (`No account found` `It looks like there isn't an account on Cloud Native Computing Foundation tied to this email address`)

Reply:
I verified it and it is indeed true.